README-EN.md

```diff
@@ -128,30 +150,27 @@ If the project need to be proguarded, put the following code into the proguard.p
 -keep class * implements com.playableads.MultiPlayLoadingListener {*;}
 -keep class * implements com.playableads.MultiPlayPreloadingListener {*;}
 -keep class com.playableads.PlayableAds {
-    public void onDestroy();
     public static com.playableads.PlayableAds getInstance();
-    public void requestPlayableAds(com.playableads.PlayPreloadingListener, java.lang.String);
-    public void requestPlayableAds(java.lang.String, com.playableads.PlayPreloadingListener);
     public synchronized static com.playableads.PlayableAds init(android.content.Context, java.lang.String);
-    public void presentPlayableAD(java.lang.String, com.playableads.PlayLoadingListener);
-    public void presentPlayableAd(com.playableads.PlayLoadingListener);
-    public boolean canPresentAd(java.lang.String);
-    public void setMultiLoadingListener(com.playableads.MultiPlayLoadingListener);
-    public void setMultiPreloadingListener(com.playableads.MultiPlayPreloadingListener);
-    public void setCacheCountPerUnitId(int);
-    public void setAutoLoadAd(boolean);
+    public <methods>;
 }
 ```
 
-# 5 Code Sample
-Click [HERE](https://github.com/zplayads/PlayableAdsDemo-android) to download Demo
+## * state code and description
+
+|state code|description|
+|-----|----|
+|1001|request constructed error|
+|1002|request parameters error.|
+|1003|lack of WRITE_EXTERNAL_STORAGE|
+|1004|lack of READ_PHONE_STATE|
+|2001|payload has been loaded|
+|2002|preload finished|
+|2004|ads has filled|
+|2005|no ad|
+|5001|context is null|
+|5002|network error|
+
 
-# 6 Notes
-## 6.1 Request Ads ASAP
-To ensure the ad resource can be successfully loaded, it’s encouraged to request ads as soon as possible.
-## 6.2 Permissions
-Make sure your app was granted Phone State permission and Storage Permission, otherwise there may be no ads in your app.
-## 6.3 Request Next Ad
-* PlayableAds sdk will autoload next ad by default, when it failed to load ad they will try again 5 seconds later. You can forbiden autoload action by calling setAutoLoadAd(false).
+If you have any problems during the process of access, or if you found any issue in the SDK, please feel free to ask questions. Please refer to [here](https://help.github.com/articles/creating-an-issue). We will deal with your questions in the first place and we are very grateful.
```

Comment:
### Note
If you have any problems during the process of access, or if you found any issue in the SDK, please feel free to ask questions. Please refer to [here](https://help.github.com/articles/creating-an-issue).